src/ontology/edits/oeo-model.omn

```diff
@@ -549,6 +561,75 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1944",
         rdfs:label "has time stamp alignment"@en
     
     
+ObjectProperty: OEO_00390071
+
+    Annotations: 
+        OEO_00020426 "add
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/2013
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/2020",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "has study descriptor tag is a relation between a scenario bundle and a oeo class represented by a study descriptor that is tagging it.
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "has study descriptor tag is a relation between a scenario bundle and an oeo class which is used as a study descriptor.",
```

Reply:
I'd propose to add an elucidation annotation with a text like that:
"OEO classes that are used as study descriptors are annotated by the annotation property "oekg annotation" with value "study descriptor".